### ISSUE TITLE
Support for Play 2.7

### ISSUE THREAD

**Issue by apflieger** [gradle/playframework#92]
Using

```
play {
    platform {
        playVersion = '2.7.0'
    }
}
```

I get `Not a supported Play version: 2.7.0. This plugin is compatible with: [PLAY_2_3_X, PLAY_2_4_X, PLAY_2_5_X, PLAY_2_6_X].`

:)

**Comment by bmuschko** [comment]
The currently supported Play versions are [documented in the user guide](https://gradle.github.io/playframework/#play_framework). Play 2.7 is not supported yet. I am hoping that we can add support for newer Play versions as soon as we have more adopters for this plugin.

**Comment by apflieger** [comment]
To be sure, there's no way to build Play 2.7 on gradle right now, right? I tried the legacy pluginwhich fails with the same error

**Comment by bmuschko** [comment]
That's correct. The old and the new plugin have identical feature sets at the moment. The main difference is:

* old plugin: uses the software model and lives in Gradle core
* new plugin: does not use the software model and is distributed via the Gradle plugin portal

**Comment by aadrian** [comment]
Any news on this issue? 

( 2.8.0 is already around the corner https://github.com/playframework/playframework/releases/tag/2.8.0-M3 )

Thank you.

**Comment by bmuschko** [comment]
@big-guy Can you comment on the plans for this?

**Comment by arjunchhabra** [comment]
Hi all. Any updates on this item? @big-guy / @bmuschko

**Comment by wooyea** [comment]
> Hi all. Any updates on this item? @big-guy / @bmuschko

latest snapshot works for me

**Comment by aadrian** [comment]
@wooyea does it also work with 2.8.0 ?

**Comment by wooyea** [comment]
> @wooyea does it also work with 2.8.0 ?

Probably not. Supported version prefix was hardcoded.

**Comment by aadrian** [comment]
@wooyea thanks for the info.

It's a pity since Play 2.8 has just been released :( .

**Comment by ricardo-s-ferreira** [comment]
Do you have any forecast for release date?

**Comment by hallyhaa** [comment]
> [...] I am hoping that we can add support for newer Play versions as soon as we have more adopters for this plugin.

I think it's the other way around: you'll get more adopters as soon as the plugin supports the newest play versions. (I typically would like to test it with the newest play version before even thinking of using it anyware.)

Vicious circle, this?

**Comment by apflieger** [comment]
Here I even considered downgrading Play to use this plugin

**Comment by ricardo-s-ferreira** [comment]
Hi,
I just need the current version to be released. I couldn't be locked in
2.6. So if you have a target date I could evaluate if not I've to find
alternatives.



Arnaud Pflieger <notifications@github.com> escreveu no dia quarta,
5/02/2020 à(s) 07:51:

**Comment by polentino** [comment]
> The currently supported Play versions are [documented in the user guide](https://gradle.github.io/playframework/#play_framework). Play 2.7 is not supported yet. I am hoping that we can add support for newer Play versions as soon as we have more adopters for this plugin.

Given that's been more than a year since Play 2.7.0 was first released, and now there's even 2.8 available, would it be possible to update the plugin to support 2.7?

**Comment by ricardo-s-ferreira** [comment]
No one will adopters this since doesn't support 2.8. who will start using this with play 2.5, or even 2.6 not fully supported?
I just remove from our project.
RIP.

**Comment by benmccann** [comment]
I sent a PR for Play 2.8 support: https://github.com/gradle/playframework/pull/133

**Comment by yfeleke** [comment]
I have spent the majority of the last 3 hours trying to keep Gradle, how is this still not solved? We need:
- Code samples: Play 2.8 + Gradle plugin (in the current non-software model)
- Latest Gradle v7 is coming + Gradle plugin + Java 14 should all play nice together until play framework starts supporting java 14
It is only my spite for sbt that is keeping me here, even the play docs are starting to only use sbt.

**Comment by polentino** [comment]
> It is only my spite for sbt that is keeping me here, even the play docs are starting to only use sbt.
same here. 

@bmuschko / @big-guy any idea if support for 2.7 will ever land? At some point, we devs would like to have a definitive [yes, no] answer

**Comment by JLLeitschuh** [comment]
I'm happy to merge changes once #136 is resolved. Currently, I can't really trust the CI to verify if a change will break anything. Once I have confidence in CI, I'll be more confident merging external PRs.

**Comment by bmuschko** [comment]
@polentino Unfortunately, I am not involved in the project anymore. Cheers!

**Comment by JLLeitschuh** [comment]
Support is requested on #136. If you'd like to see this issue resolved, please do help out with resolving #136.

**Comment by cstroe** [comment]
See https://github.com/gradle/playframework/pull/147 for a fix to #136

**Comment by cstroe** [comment]
Now that #147 is in master, we should merge master into this branch and get a build going.

**Comment by JLLeitschuh** [comment]
This has been resolved in v0.10

https://gradle.github.io/playframework/#change_log
